certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13162 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "EasyVista Service Manager Modification Décembre 2022",
```

Comment:
@ezv-compteintegration I would highly recommend a better description for the connector in English as this will show in the connector [documentation](https://learn.microsoft.com/en-us/connectors/easyvista/).